platform/security/rest/security-rest-cxfwrapper/src/main/java/org/codice/ddf/cxf/oauth/OAuthSecurityImpl.java

```diff
@@ -326,7 +326,12 @@ private String getNewAccessToken(
     }
 
     if (response.getStatus() != HttpStatus.SC_OK) {
-      LOGGER.debug("Unable to retrieve system access token. {}", body);
+      LOGGER.debug(
```

Comment:
❔ How does this relate to removing the karaf feature?

Reply:
oops didn't intend for that to be a part of this PR